.github/workflows/deploy.yaml

```diff
@@ -70,117 +70,136 @@ jobs:
     uses: "./.github/workflows/aws-deploy.yaml"
     with:
       role-to-assume: "arn:aws:iam::531805629419:role/sagebase-github-oidc-sage-bionetworks-it"
+      role-session-name: ${{ steps.role-session-name.outputs.substring }}
```

Comment:
can you reference a `step` from a different job?  i think `step` here is not in the same scope as the one that's in `org-formation` job.  you probably need to add that `Get role session name` to every one of these jobs?

Reply:
Yes, you are right.  I have to find a way to pass the value between jobs.

Reply:
Done!